.github/workflows/build-and-review-pr.yml

```diff
@@ -60,3 +60,293 @@ jobs:
+
+      - name: And the other outputs should indicate a failure happened
+        if: always()
+        run: |
+          ./test/assert-values-match.sh --name "test-outcome output" --expected "Failed" --actual "${{ steps.missing-runinfo.outputs.test-outcome }}"
+          ./test/assert-value-is-empty.sh --name "test-results-file-path output" --value "${{ steps.missing-runinfo.outputs.test-results-file-path }}"
+          ./test/assert-value-is-empty.sh --name "trx-files output" --value "${{ steps.missing-runinfo.outputs.trx-files }}"
+          ./test/assert-value-is-empty.sh --name "test-results-truncated output" --value "${{ steps.missing-runinfo.outputs.test-results-truncated }}"
+
+      
+      
+      
+      
+      # Conditions:
+      # empty TestRun.Results && TestRun.TestDefinitions  (format is auto-fixed && report title set to _computerName)
+      # empty TestRun.Results.UnitTestResult &&TestRun.TestDefinitions.UnitTest (format is auto-fixed)
+      # present but single entry for TestRun.Results.UnitTestResult &&TestRun.TestDefinitions.UnitTest (format is auto-fixed)
+      # using report title filter
+      # no report title filter && defaulting to dll name
+      # has a failing test && create-status-check=true && ignoreTestFailures=false  (test-outcome output = Failed, status check=failure)
+      # has a failing test && create-status-check=true && ignoreTestFailures=true  (test-outcome output = Failed, status check=neutral)
+      # no failing tests && create-status-check=true (test-outcome output = Passed, status check=success)
+      # creating status check (one per trx)
+      # creating pr comment with normal message && update-comment-if-one-exists=true (test-outcome-truncated output = false)
+      # creating pr comment with normal message && update-comment-if-one-exists=false (test-outcome-truncated output = false)
+      # creating pr comment with truncated message && new comment-identifier (test-outcome-truncated output = true)
+
+      # creating results file - Use this to determine the actual output
+
+      #---------------------------------------------------------------------------------------------
+      #-----------------------------------------
+      # TEST -
+      #-----------------------------------------
+      # - name: '-------------------------------------------------------------------------------------------------------'
+      #   run: echo ""
+
+      # - name: When process-dotnet-test-results is called with
+      #   id: change-me
+      #   uses: ./
+      #   with:
+      #     github-token: '${{ secrets.GITHUB_TOKEN }}'
+      #     base-directory: '${{ env. }}'
+      #     ignore-test-failures: false
+      #     create-status-check: true
+      #     create-pr-comment: true
+      #     create-results-file: true
+      #     update-comment-if-one-exists: true
+      #     timezone: 'America/Denver'
+      #     comment-identifier: 'dotnet-test-results'
+      #     report-title-filter: '?'
```

Comment:
Out of curiosity, what is the reason for keeping this commented out stuff around?

Reply:
I haven't finished with the implementation of these tests yet.  It's just an easy copy/paste for each scenario so I don't have to keep typing out the action with all of its inputs.